### ISSUE TITLE
Using notes: Cannot create event: Bad request

### ISSUE THREAD

**Issue by albsch** [nibdo/bloben-app#54]
Trying to paste a big text into a note causes an error when you want to create a new event:

```
{"level":"error","message":"Status: 400 Message: Bad Request: null","tags":["CALDAV","REST"],"timestamp":"2022-07-16 10:59:58"}
```

The error happens when I try to paste this text into the `notes` field:

```
TEST testest test TESt tes testset

17.08.2022

Test Testet test Test tet test

Teste test Teste test testetset testetet Test-Teste.  Test teste tes test, teste tetest tsetset tes te Teste tets test te Testet te Test tet Test tes teste.  Test test 4 Teset test-Tetsetset test test tsets tsets test tset tset Test-Test testet.

Test Teste test.test.test teste test.test.tset 
```

**Comment by palitu** [comment]
I think i have the same issue, though not so big.  i found it was if there was a new line added.

see https://github.com/nibdo/bloben-app/issues/57

**Comment by nibdo** [comment]
Thank you both for pointing this, it is problem with ical parser.

**Comment by nibdo** [comment]
Fixed in 0.11.0
